package.json

```diff
@@ -92,7 +94,15 @@
     "gulp-debug": "*",
     "gulp-git": "*",
     "gulp-tap": "*",
-    "merge-stream": "*"
+    "jasmine-core": "^2.8.0",
```

Comment:
jasmine-core is probably not needed since `karma-jasmine` is so old and bundles it internally.